elasticsearch/plan.sh

```diff
@@ -6,25 +6,28 @@ pkg_description="Open Source, Distributed, RESTful Search Engine"
 pkg_upstream_url="https://elastic.co"
 pkg_license=('Revised BSD')
 pkg_source=https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/${pkg_version}/${pkg_name}-${pkg_version}.tar.gz
-pkg_filename=${pkg_name}-${pkg_version}.tar.gz
 pkg_shasum=23a369ef42955c19aaaf9e34891eea3a055ed217d7fbe76da0998a7a54bbe167
-pkg_deps=(core/glibc core/jre8)
+pkg_deps=(
+  core/busybox-static
```

Comment:
It's for `wget` and `awk` in the health check, and the elasticsearch and plugin scripts in the bin dir use things like `dirname`, `hostname`, etc.